pkgs/development/python-modules/cebra/default.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  fetchPypi,
+  pythonOlder,
+  python3Packages,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "cebra";
+  version = "0.5.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-wNTuK5i9VJvRQg10G1xtzf4bMIaRqdOQzTD4P08viZw=";
+  };
+
+  disabled = pythonOlder "3.9";
+
+  # not checking for deps since it fails to test without literate_dataclasses
+  doCheck = false;
+  dontCheckRuntimeDeps = true;
+
+  propagateBuildInputs = with python3Packages; [
```

```suggestion
  dependencies = [
```

And list all the dependencies at the top.